application/classes/Ushahidi/Core.php

```diff
@@ -542,6 +542,34 @@ public static function init()
 		$di->setter['Ushahidi_Listener_PostSetListener']['setRepo'] =
 			$di->lazyGet('repository.notification.queue');
 
+		// Login rate limiter
+		$di->set('ratelimiter.login', $di->lazyNew('BehEh\Flaps\Flap'));
+
+		$di->params['BehEh\Flaps\Flap'] = [
+			'storage' => $di->lazyGet('ratelimiter.storage'),
+			'name' => 'login'
+		];
+
+		// Rate limiter storage
+		$di->set('ratelimiter.storage', $di->lazyNew('BehEh\Flaps\Storage\DoctrineCacheAdapter'));
```

Comment:
Not sure if we want to to track login counts in the DB. We could implement a DB adapter but I think something like memcache makes more sense for a production environment. I suppose we can make memcache and the FilesystemCache available as config options?